src/assets/drizzle/scripts/drizzle.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+import 'prismjs';
+
+const dom = {};
+
+dom.nav = document.getElementById('nav');
+dom.navMenu = document.getElementById('nav-menu');
+dom.navToggle = dom.nav.querySelector('a[href="#nav"]');
+dom.navLinks = dom.navMenu.querySelectorAll('a');
+
+function setActiveNavItem (pathname) {
```

Comment:
Should we make an issue to create a helper or something to allow us to bake this into the nav, making it more about the current _page_ than the current hash? Just a thought!